class-tapi-query.php

```diff
@@ -59,6 +59,11 @@ private function _resolve_type() {
 
 	public function query() {
 		$this->tweets = WP_Twitter_API()->get( $this->path, $this->query_args );
+		if ( is_object( $this->tweets ) ) {
+			if ( ! empty( $this->tweets->statuses ) ){
+				$this->tweets = $this->tweets->statuses;
+			}
+		}
 		if ( is_array( $this->tweets ) && ! empty( $this->tweets ) && ! isset( $this->tweets->errors ) ) {
```

Comment:
Should this be `elseif`? obviously these two conditions can't both be true so it's not a big deal, but maybe worth changing for clarity.